angelsbioprocessing/changelog.txt

```diff
@@ -1,4 +1,14 @@
 ---------------------------------------------------------------------------------------------------
+Version: 0.7.10
+Date: ???
+  Changes:
+    - Crystal powder liquifying now required purified water similar how to dus requires acid (270)

```

Comment:
how dust requires?

Reply:
resolved in https://github.com/Arch666Angel/mods/pull/272/commits/44614af7c5a752ceef52f1850ce2f26d55e2554f